__tests__/react/upload.js

```diff
@@ -86,14 +86,6 @@ describe("userEvent.upload", () => {
     expect(input.files[0]).toStrictEqual(file);
     expect(input.files.item(0)).toStrictEqual(file);
     expect(input.files).toHaveLength(1);
-
-    fireEvent.change(input, {
-      target: { files: { item: () => {}, length: 0 } },
-    });
-
-    expect(input.files[0]).toBeUndefined();
```

Comment:
This tests do nothing special to `user-event` library. They only checks that `fireEvent` function works properly.